src/analysis/special_functions/exp_log.lean

```diff
@@ -475,6 +475,20 @@ end
 lemma log_nonpos (hx : 0 ≤ x) (h'x : x ≤ 1) : log x ≤ 0 :=
 (log_nonpos_iff' hx).2 h'x
 
+lemma log_nonzero_of_ne_one (x: ℝ) (hx_pos: 0 < x) (hx: x ≠ 1): real.log x ≠ 0 :=
```

Comment:
Note that mathlib puts a space before the colon (also below):

```suggestion
lemma log_nonzero_of_ne_one (x : ℝ) (hx_pos : 0 < x) (hx : x ≠ 1): real.log x ≠ 0 :=
```

Reply:
Forgot about that, my bad, thanks.